packages/tools/kadena-cli/src/tx/utils/txHelpers.ts

```diff
@@ -62,12 +62,14 @@ export function getSignersStatus(
 /**
  * Retrieves all transaction file names from the transaction directory based on the signature status.
  * @param {boolean} signed - Whether to retrieve signed or unsigned transactions.
+ * @param {boolean} all - Whether to retrieve all transactions (signed and unsigned).
  * @returns {Promise<string[]>} A promise that resolves to an array of transaction file names.
  * @throws Throws an error if reading the transaction directory fails.
  */
 export async function getTransactions(
   signed: boolean,
   directory: string,
+  all: boolean = false,
```

Comment:
I thought make it as a short term fix for now! maybe will fix it properly after the demo